### ISSUE TITLE
[!!] please add WCAG-EM

### ISSUE THREAD

**Issue by nitedog** [w3c/wai-std-gl-overview#4]
Tricky because this page is primarily for RECs, but we also list UAAG 2.0 Note and WAI-ARIA Authoring Practices, so I think also WCAG-EM should be listed on this page.

**Comment by slhenry** [comment]
done
